spec/injector_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'injector'
+
+describe Array do
+
+context 'dave_inject' do
+  it 'addition injection' do
+  expect([1,2,3,4,5].dave_inject(0){|a, b| a + b}).to eq 15
+  end
+
+  it 'addition injection' do
+  expect([1,2,3,4,5].dave_inject{|a, b| a + b}).to eq 15
+  end
+
+  it 'subtraction injection' do
+  expect([1,2,3,4].dave_inject(12){|a, b| a - b}).to eq 2
```

Comment:
Use 2 (not 0) spaces for indentation.<br>Space missing after comma.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.